src/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import addons from '@storybook/addons'
+import WrapStyledComponentsThemePicker from './WrapStory'
+
+const wrapperTheme = (themes) => {
+  const channel = addons.getChannel();
+  return (storyFn, context) => (
+    <WrapStyledComponentsThemePicker
+      themes={themes}
+      channel={channel}
+      >{storyFn(context)}
+    </WrapStyledComponentsThemePicker>
+  );
+}
+
+export function withThemes(themes, defaultTheme) {
+  return (storyFn, context) => wrapperTheme(themes, defaultTheme)(storyFn, context)
+}
```

Comment:
how about cleaning this up a little? like
```javascript
export const withThemes = themes => (storyFn, context) => {
  const channel = addons.getChannel();
  return (
    <WrapStyledComponentsThemePicker
      themes={themes}
      channel={channel}
      >{storyFn(context)}
    </WrapStyledComponentsThemePicker>
  )
}
```

Reply:
I don't have the project with me at this right moment, but I remember that I tried that approach before and somehow it failed. 
However like the PR states, a lot of things changed. 
I can give it another try and if it works, I'm pretty ok with the change. 👍